README.md

```diff
@@ -20,7 +20,7 @@
 Z-Wave is an amazing piece of technology, but it is difficult to get started with and develop for.  
 **It shouldn't be!** We've got you covered.
 
-Built on Node.js, **Z-Wave JS** runs on almost anything with a little bit of computing power and a serial port. Check out the [documentation](https://zwave-js.github.io/node-zwave-js) to get started!
+Built on Node.js and compatible with all known Z-Wave controller hardware adapters, **Z-Wave JS** runs on almost anything with a little bit of computing power and a serial port. Check out the [documentation](https://zwave-js.github.io/node-zwave-js) to get started!
```

Comment:
I think we need to relativize a bit here. 500-series and Z-Wave 700 (yeah, they changed the name) controllers are supported - I won't make such promises for the 300 series, although they probably work too.

Reply:
OK changed to "compatible with all known Z-Wave 500 series + Z-Wave 700 series controller hardware adapters"